.pre-commit-config.yaml

```diff
@@ -15,21 +15,18 @@ repos:
       - id: check-yaml
       - id: pretty-format-json
 
-  - repo: https://github.com/pre-commit/mirrors-yapf
-    rev: v0.31.0
+  - repo: https://github.com/psf/black
+    rev: 21.5b1
     hooks:
-      - id: yapf
-        name: formatting
-        entry: yapf
-        args: [--parallel]
-        language: python
-        types: [python]
+      - id: black
+        args: [
+            --line-length=120,
+            --skip-string-normalization,
+            --skip-magic-trailing-comma,
```

Comment:
can we add this to some config file so anyone who would run black separately gets the same result?

Reply:
yep I can put this into a `pyproject.toml` (the only configuration that black supports)